src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -510,8 +510,8 @@ namespace HVACUnitarySystem {
         // Uses the status flags to trigger initializations.
```

Comment:
Massive diff's here due to clang.format. Will add comments at the few places changes were actually made.